src/smlVersion.cpp

```diff
@@ -10,9 +12,9 @@
 #include <filesystem>
 #include "picosha2.h"
 
-#ifdef _WIN32
```

Comment:
I think this should be kept because dlls are only on Windows and the paths are only correct for Windows, hashes are of dlls as well